features/terraform_aws_example.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Terraform AWS Example
```

Comment:
Is putting these in a `features` folder a BDD idiom? Or a requirement? 

If it's not a requirement, I'd be tempted to put them under `test/features` instead...